lib/helpers/path-to-eslint.js

```diff
@@ -0,0 +1,20 @@
+/*eslint-env node*/
+
+'use strict';
+
+var join = require('path').join;
+var asyncExec = require('./async-exec');
+
+/**
+ * Get the path to the ESLint executable in the dependencies
+ *
+ * @returns {Promise->String} the path to the ESLint executable
+ */
+module.exports = function() {
+  return asyncExec('npm', ['bin'])
+    .then(function(location) {
+      return join(
+        location.trim(), '../ember-cli-eslint/node_modules/.bin/eslint'
```

Comment:
Is there a better way to resolve the location of the `eslint` binary that's installed with this package?  I want to make sure that we use the right one so that the same version is used between `ember test` and `ember lint`